### PR TITLE
Add Galician translations

### PR DESCRIPTION
**Pull request by frandieguez**
Please add latest galician translations

### CODE REVIEW

File: po/gl.po [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+msgstr "Bucar aplicativos compatíbeis en Software"
+
+#: src/filechooser.c:363
+msgid "_Open"
+msgstr "_Abrir"
+
+#: src/screenshotdialog.c:93
+#, c-format
+msgid "Share this screenshot with %s?"
+msgstr "Compartir esta captura de pantalal con %s?"
```
Comment (mariospr) [inline po/gl.po]: pantalal -> pantalla

**Comment by frandieguez** [comment]
Typo fixed! Thanks for the help!

**Comment by mariospr** [comment]
Thanks for the quick response. Btw, I think it would be nice if you could squash the fixup commits onto the initial one adding the translation, so that only one commit gets pushed with no typos. What do you think?

**Comment by frandieguez** [comment]
Sure. Commits squashed as requested. This way is much cleaner, at least for the initial commit. ;)

Sidenote: I think I'll send another pull request to flatpak to merge its Galician translations by tomorrow

**Pull request merged**